facenet/FaceDetector.py

```diff
@@ -0,0 +1,20 @@
+import cv2
+
+# HaarCascade Face Detector
+class faceDetector: 
```

Comment:
use PascalCase for class names `FaceDetector`

Reply:
Changed it